src/web_ui/__init__.py

```diff
@@ -100,7 +113,7 @@ def realtime_readings():
     )
 
 
-@app.route("/network-scan")
+@ app.route("/network-scan")
```

Comment:
Looks like these spaces snuck in, maybe during merge conflict resolution?

Reply:
Yep, thanks for spotting it!